pkgs/development/libraries/exiv2/default.nix

```diff
@@ -58,8 +53,6 @@ stdenv.mkDerivation rec {
     "doc"
   ];
 
-  doCheck = stdenv.isLinux;
```

Comment:
`doCheck` is default false so this disabled them.

Reply:
Oh :(